uint8 token
float32 panSpeed
float32 tiltSpeed
float32 zoomSpeed
---
int32 status